src/components/CommunityModuleBadge.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import {StyleSheet, View, Text} from 'react-native';
+import {useTheme} from '@react-navigation/native';
+import {SymbolIcon} from 'react-native-xaml';
+
+const createStyles = (colors: any) =>
+  StyleSheet.create({
+    badgeContainer: {
+      height: 28,
+      width: 190,
+      borderRadius: 2,
+      flexDirection: 'row',
+      alignItems: 'center',
+      justifyContent: 'center',
+      backgroundColor: colors.text,
+      margin: 2,
+    },
+    badgeText: {
+      fontSize: 15,
+      fontWeight: '400',
+      color: 'white',
+      paddingRight: 4,
+    },
+  });
+
+export function CommunityModuleBadge() {
+  const {colors} = useTheme();
+  const styles = createStyles(colors);
+  return (
+    <View
+      style={styles.badgeContainer}
+      tooltip="This component is not a part of React Native Windows core. The package for this component must be explicitly added in order to have access to this component in a React Native Windows project.">
```

Comment:
consider removing the mention to RNW for a non-core component since the same woudl be true of e.g. android :)
```suggestion
      tooltip="This component is not a part of React Native core. The package for this component must be explicitly added in order to have access to this component in a React Native Windows project.">
```